### DISCUSSION TITLE
Set the default time format (12H or 24H) in an environment variable

### DISCUSSION THREAD

**Discussion by borjarobert** [lukevella/rallly#695] [Ideas]
For some reason, some of my users are getting 12H time format as default, which is hard for them to understand for noon as we're used to 24H.

I'd love to be able to select a default time format through an environment variable.

Thanks in advance and best regards. Rallly has been working flawlessly!

**Comment by lukevella** [comment]
Hey @borjarobert, thanks for the feedback. I understand the frustration. Rather than offloading this problem to the administrator and adding more configuration options which is only useful for self-hosters this will be addressed by improving Rallly's ability to guess the user's locale preferences and making it easier for participants to switch between 12/24 hour time. Thanks for raising.

**Reply by borjarobert** [reply]
Thanks! Sounds like a great solution to me.

All the best!

**Reply by adan-ea** [reply]
Hi there, any updates on that ? I'm still getting a default 12h format on multiple device that has their locale to a 24h format.
I've seen the easy way of switching tho so that's nice! 
Thanks :]

**Comment by tuxcrafter** [comment]
I would like a default "Time format" setting as variable in config.env so the default can be set to 24H for all new registered users.

![image](https://github.com/user-attachments/assets/8ad772d9-94c0-4fd2-b01a-d13b15670fc6)
